install/kubernetes/helm/istio-init/files/crd-10.yaml

```diff
@@ -121,6 +131,8 @@ spec:
     kind: ClusterRbacConfig
     plural: clusterrbacconfigs
     singular: clusterrbacconfig
+    shortNames:
+    - crbaccfg
```

Comment:
this one is too complicated.

Reply:
For now, can we just use `rbac` for this? 
/cc @liminw